tests/StateObserver.py

```diff
@@ -10,31 +10,60 @@
 import time
 
 class State():
-    def __init__(self, addr, assetId):       
-        self.balance =  OnyxCashWrapper.Balance(addr) if assetId is "OnyxCash" else AssetsWrapper.GetBalance(addr, assetId)            
+    def __init__(self, addr, targetAssets):           
+        self.addr = addr
+        self.balances = dict()
+        self.totalProfits = dict()
+        self.fiatAmounts = dict()
+        for assetId in targetAssets:
+            self.balances[assetId] = OnyxCashWrapper.Balance(addr) if assetId is "OnyxCash" else AssetsWrapper.GetBalance(addr, assetId)   
+            self.totalProfits[assetId] = InternalRevenueServiceWrapper.GetTotalReward(assetId, addr)
+            self.fiatAmounts[assetId] = ExchangeWrapper.GetFiatAmount(addr, assetId)
+
         self.rejectationCounter = RequestHolderWrapper.RejectationCounter(addr)
-        self.totalProfit = InternalRevenueServiceWrapper.GetTotalReward(assetId, addr)
         self.successfulOperations = RequestHolderWrapper.GetSuccessfulOperationsCount(addr)
         self.failedOperations = RequestHolderWrapper.GetFailedOperationsCount(addr)
         self.isBlocked = OnyxPayWrapper.IsBlockedUser(addr)
-        self.fiatAmount = ExchangeWrapper.GetFiatAmount(addr, assetId)
 
+    def PrintState(self):
+        print(self.addr)
+        print("Balances:")
+        for i, asset in enumerate(self.balances):  
```

Comment:
i and enumerate() aren't used here, also it's better to use .items()
```suggestion
        for asset, balance in self.balances.items():
```

Reply:
+